server/routes/api/section.js

```diff
@@ -92,9 +99,8 @@ router.post('/createSection', (req, res) => {
 
 });
 
-
-//currently unfinished
-router.get('/updateSection', (req, res) => {
+// updates a student section
+router.post('/updateSection', (req, res) => {
```

Comment:
since this is an update, this should be `router.put`

Reply:
```suggestion
router.put('/updateSection', (req, res) => {
```

Reply:
Done!